### DISCUSSION TITLE
Create a PKG directly run the app to erase/install the machine

### DISCUSSION THREAD

**Discussion by mani2care** [grahampugh/erase-install#108] [Ideas]
Create a PKG directly run the app to erase/install the machine without jamf and also after installed the OS is that possible to automate creating the user account ?

**Comment by grahampugh** [comment]
Hi @mani2care . My workflow is to install the package provided in this repository and use Jamf's Run Command to execute the appropriate command.  I will not be providing an automated package since there are infinite parameters that would have to be provided, meaning infinite packages. But you can do that pretty easily yourself with a post install script if you want. 

Again, there are existing open source projects that can create user accounts. You may want to look at pycreateuser for example. Or use DEP to create the account. I will not be adding this functionality to erase-install.
